playbooks/portals-setup-initial.yml

```diff
@@ -24,17 +24,15 @@
   any_errors_fatal: True
   vars:
     max_hosts: "{{ groups['webportals'] | length - 1 }}"
-    lastpass_required: True
-    lastpass_allow_missing_user_credentials: True
   vars_files:
     - "{{ common_vars_file }}"
     - "{{ custom_vars_file  }}"
   tasks:
     - name: Fail if you are targeting all dev and prod webportals
       include_tasks: tasks/host-limit-check.yml
 
-    - name: Include creating webportal user credentials in LastPass (if missing)
-      include_tasks: tasks/lastpass-save-user-credentials.yml
+    - name: Include creating OS user credentials in secrets storage (if missing)
+      include_tasks: "tasks/secrets-storage-common-save-user-credentials.yml"
```

Comment:
why does this task need quotation marks?

Reply:
Removed quotes.
It was leftover from refactorings.